test/tools/utils.ts

```diff
@@ -472,3 +472,27 @@ export const sorted = <T>(iterable: Iterable<T>, how: (a: T, b: T) => 0 | 1 | -1
   items.sort(how);
   return items;
 };
+
+/**
+ * This helper method is necessary as the changes in NODE-4720 required the Topology constructor to
+ * accept a client with access to a logger. This helper serves to keep old tests working with minimal
+ * changes*/
+export function topologyWithPlaceholderClient(
```

Comment:
How come we decided to stub out a client instead of just creating a client inline here?

Reply:
This is only ever used in unit tests, so constructing a client felt like overkill.

Reply:
I think we would wind up with less code, is there an issue with: `new MongoClient('mongodb://iLoveJavaScript')` as in will it fail for some reason? I am assuming we wouldn't reach connect logic in unit tests